GMG/Core/Models/ScorePlayer/ScorePlayer.swift

```diff
@@ -117,6 +114,11 @@ final class ScorePlayer {
                 do {
                     try self.activateAudioSession()
                     try self.engine.start()
+
```

Comment:
재생 중 오디오 세션이 변경되면 소리가 출력되지 않는 문제가 있습니다.
오디오 세션 활성화 시점과 엔진 재시작 타이밍이 실제 재생 타이밍과 겹치면서 발생하는 문제로 보입니다.

오디오 세션 변경 시 재생 상태를 잠시 pause하는 처리가 있으면 좋을 것 같아요.

버그 재현 방법:
ChordProgressView에서 스피커로 재생 중인 상태에서 이어폰을 연결하면 코드 및 허밍 소리가 출력되지 않습니다.

```suggestion
self.pause()
```

Reply:
`seek` 메서드처럼 이전 재생 상태에 따라 재생하도록 수정했어요~